Directory.Packages.props

```diff
@@ -1,37 +1,37 @@
 <Project>
   <ItemGroup>
-    <PackageVersion Include="Autofac.Extensions.DependencyInjection" Version="4.2.2" />
+    <PackageVersion Include="Autofac.Extensions.DependencyInjection" Version="10.0.0" />
     <PackageVersion Include="BenchmarkDotNet" Version="0.14.0" />
     <PackageVersion Include="coverlet.msbuild" Version="6.0.2" />
     <PackageVersion Include="GitHubActionsTestLogger" Version="2.4.1" />
     <PackageVersion Include="IdentityServer4" Version="3.1.4" />
     <PackageVersion Include="IdentityServer4.AccessTokenValidation" Version="3.0.1" />
     <PackageVersion Include="Microsoft.AspNetCore" Version="2.1.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.ApiExplorer" Version="2.1.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="2.1.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Formatters.Json" Version="2.1.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.ApiExplorer" Version="2.1.3" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="2.1.3" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Formatters.Json" Version="2.1.18" />
```

Comment:
```suggestion
    <PackageVersion Include="Microsoft.AspNetCore.Mvc.ApiExplorer" Version="2.1.0" />
    <PackageVersion Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="2.1.0" />
    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Formatters.Json" Version="2.1.0" />
```

Reply:
@martincostello: Why we don't want to have latest 2.1.x versions here? They are not marked as deprecated in nuget.org: https://www.nuget.org/packages/Microsoft.AspNetCore.StaticFiles

Reply:
It's not Swashbuckle's business to force specific versions of dependencies onto our consumers - applications who want the latest versions can update their own application to do so. It makes reasoning about compatibility for use with the hundreds of projects who consume our projects much easier to deal with.

Just the test-dependencies that dependabot is failing to update for us (in an idea world this PR wouldn't be needed at all) are what should be updated.